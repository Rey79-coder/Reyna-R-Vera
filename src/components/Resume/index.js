import React from 'react';
import resumeImage from "../../assets/resume/resume-image.png";

function Resume() {
  return (
<section className="my-5">

<button>
<a id="resume" href="https://drive.google.com/file/d/1gOhCSv_aArqI2zmdvf0uQGZld0rRFNkG/view?usp=sharing"
        alt="View complete resume" class="text-email" target="_blank" rel="noreferrer">
  <img 
  href="#resume"
  src={resumeImage} 
  className="my-2" 
  style={{ width: "100%" }} 
  alt="resume" />
  </a>
  </button>
  
      
</section>
  );
}

export default Resume;