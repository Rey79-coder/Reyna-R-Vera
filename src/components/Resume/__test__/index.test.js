import React from 'react';

import { render, cleanup } from '@testing-library/react';

import '@testing-library/jest-dom/extend-expect';

import Resume from '..';

afterEach(cleanup);


describe('Resume component', () => {
    // First Test
    it('renders', () => {
        render(<Resume />);
      });
  
    // Second Test
    it('matches snapshot DOM node structure', () => {
        // render Resume
        const { asFragment } = render(<Resume />);
        expect(asFragment()).toMatchSnapshot();
      });

    
  })