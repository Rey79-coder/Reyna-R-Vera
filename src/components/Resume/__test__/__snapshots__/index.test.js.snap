// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <button>
      <a
        alt="View complete resume"
        class="text-email"
        href="https://drive.google.com/file/d/1gOhCSv_aArqI2zmdvf0uQGZld0rRFNkG/view?usp=sharing"
        id="resume"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="resume"
          class="my-2"
          href="#resume"
          src="resume-image.png"
          style="width: 100%;"
        />
      </a>
    </button>
  </section>
</DocumentFragment>
`;
